### PR TITLE
[DEMA-566] apply existing formula to clustered redis readme

### PR DESCRIPTION
**Pull request by MarcinGinszt**

### CODE REVIEW

File: redis-cluster/README.md [hunk 1]
```diff
@@ -1,121 +1,103 @@
...
+|-------------------------------------------------------------|--------------------------------------------------------------|--------------------|----------|------------------------|---------------------------------------|
+| Standalone                                                  | No                                                           | Yes                | No       | 1                      |                                       |
+| [Replicated](https://redis.io/docs/management/replication/) | No (replica nodes are used e.g. to optimize read operations) | Yes                | No       | 1 master, n replicas   |                                       |
+| [Sentinel](https://redis.io/docs/management/sentinel/)      | Yes (master election in case of failure)                     | Yes                | No       | 3+                     | Needs special handling on client side |
+| [Cluster](https://redis.io/docs/management/scaling/)        | Yes (master election in case of failure)                     | No                 | Yes      | 3+ masters, n replicas | Needs special handling on client side |
+
+If you are interested in using another flavour, let us know!
```
Comment (sbuliarca) [inline redis-cluster/README.md:28]: Would be great to specify here which one is supported by the other `redis` base.

File: redis/README.md
Comment (sbuliarca) [inline redis/README.md:1]: Would be great to outline the cluster alternative using the redis-cluster base

File: redis-cluster/README.md (hunk 1 above)
Comment (MarcinGinszt) [reply redis-cluster/README.md:30]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/38/commits/04c99e377901fca1664a910ce96652267528e1c6)

File: redis/README.md
Comment (MarcinGinszt) [reply redis/README.md:1]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/38/commits/04c99e377901fca1664a910ce96652267528e1c6)

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**